spec/models/category_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
+  
+  describe 'DB table' do
+    it { is_expected.to have_db_column :id }
+    it { is_expected.to have_db_column :title }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :title }
+  end
+
```

Comment:
you need to test the association between category and product in both model specs